src/components/Navbar.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { StyleSheet, View, Image } from 'react-native';
+import colors from '../assets/colors.js'
+import images from '../assets/images.js'
+
+class Navbar extends React.Component {
+  render() {
+    return (
+      <View style={styles.navbar}>
+        <View style={styles.imageContainer}>
+          <Image source={images.magGlass}/>
+        </View>
+        <View style={styles.imageContainer}>
+          <Image source={images.camera}/>
+        </View>
+        <View style={styles.imageContainer}>
+          <Image source={images.recycleBook}/>
+        </View>
+        <View style={styles.imageContainer}>
+          <Image source={images.gear}/>
+        </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  navbar: {
+    position: 'absolute', 
+    height: 85, 
+    bottom: 0, 
+    width: '100%', 
+    flexDirection: 'row', 
+    backgroundColor: colors.white,
+    alignItems: 'center',
+    justifyContent: 'space-around'
+  },
+  imageContainer: {
+    flex: .25,
```

Comment:
From docs (https://facebook.github.io/react-native/docs/layout-props#flex):
> When flex is a positive number, it makes the component flexible and it will be sized proportional to its flex value. **So a component with flex set to 2 will take twice the space as a component with flex set to 1**.

Using a decimal to represent a percentage works if the total of all flex assignments equals 1. This seemed like a good process at the time. But I'm happy to use something different if this seems unusual.